app/src/main/java/com/android/voyageur/model/user/UserRepositoryFirebase.kt

```diff
@@ -59,6 +60,41 @@ class UserRepositoryFirebase(private val db: FirebaseFirestore) : UserRepository
         .addOnFailureListener { exception -> onFailure(exception) }
   }
 
+  override fun fetchUsersByIds(
```

Comment:
please add documentation for the new functions, especially since they are part of the backend

Reply:
Sure I will add documentation for this function. As you mentioned it now could you please add documentation for the rest of the functions in this class? If I remember correctly you implemented them, right?

Reply:
I believe it was @BogdanCDev